elastic/kibana
# [ResponseOps] Granular Connector RBAC - adding back UIs for testing connectors in stack management

Part of https://github.com/elastic/kibana/issues/180908

## Summary

As part of the changes for EDR Connector Execution for Testing, this PRs updates the SentinelOne and Crowdstrike params UIs to limit to a single sub-action.

SentinelOne:
<img width="861" alt="Screenshot 2024-12-18 at 11 16 49 AM" src="https://github.com/user-attachments/assets/1a839d21-7ba7-4c4e-b3e9-8772725fad63" />

Crowdstrike:
<img width="855" alt="Screenshot 2024-12-18 at 11 17 01 AM" src="https://github.com/user-attachments/assets/b6c17964-8c49-4b5a-b1c7-bb4c6bb89621" />

### Checklist

Check the PR satisfies following conditions. 


### To test

1. In Stack Management create SentinelOne and Crowdstrike connectors (They don't need to work, you can use fake values for the url and token)
2. Test the new connectors. Verify that you only see one action type and that it can't be changed and that the params are correctly sent in the request.
